src/Common/ShardChooser/ShardChooser.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Gaming\Common\ShardChooser;
+
+final class ShardChooser
```

Comment:
Tests needed. Not done because currently the hashing functionality is hard-coded.